tests/char_edible_rating_test.cpp

```diff
@@ -501,7 +501,7 @@ TEST_CASE( "who will eat human flesh", "[will_eat][edible_rating][cannibal]" )
 
             THEN( "they can eat it without any qualms" ) {
                 expect_can_eat( dummy, flesh );
-                expect_will_eat( dummy, flesh, "", EDIBLE );
+                expect_will_eat( dummy, flesh, "Eating this raw meat probably isn't very healthy.", RAW_MEAT );
```

Comment:
You should probably replace the item with a cooked version of human flesh so this test does what it intends to do (checking that cannibals can eat human meat without problem).